kinta-lib/src/main/kotlin/com/dailymotion/kinta/integration/bitrise/Bitrise.kt

```diff
@@ -90,9 +91,8 @@ object Bitrise {
         val token_ = token ?: KintaEnv.getOrFail(KintaEnv.Var.BITRISE_PERSONAL_TOKEN)
 
         val appSlug = findAppSlug(token_, repoName)
-
         val request = Request.Builder()
-                .header("Authorization", "token $token")
+                .header("Authorization", "token $token_")
```

Comment:
Good catch! I started shadowing these local variables to make sure I don't make this mistake again.